src/vs/workbench/contrib/terminal/common/terminalStrings.ts

```diff
@@ -22,6 +22,10 @@ export const terminalStrings = {
 		value: localize('workbench.action.terminal.focus', "Focus Terminal"),
 		original: 'Focus Terminal'
 	},
+	focusAndHideAccessibleBuffer: {
+		value: localize('workbench.action.terminal.focusAndHideAccessibleBuffer', "Focus Terminal and Hide Accessible Buffer"),
+		original: 'Focus Terminal and Hide Accessible Buffer'
```

Comment:
Can this be the default behavior of the focus terminal command? As opposed to `FocusAccessibleBuffer` which focuses the accessible buffer, not the textarea?

Reply:
I tried that originally. It cannot - that does not work because `hide` has to be called on the accessible view component and we cannot access that here.

Reply:
I think you could handle that with `ITerminalInstance.onDidFocus`

Reply:
that's what I tried initially. It was confusing because the event is named `onDidRequestFocus`